modules/virt-understanding-node-labeling-obsolete-cpu-models.adoc

```diff
@@ -0,0 +1,136 @@
+// Module included in the following assemblies:
+// * virt/node_maintenance/virt-managing-node-labeling-obsolete-cpu-models.adoc
+
+[id="virt-understanding-node-labeling-obsolete-cpu-models_{context}"]
+= Understanding node labeling for obsolete CPU models
+
+To ensure that a node supports only valid CPU models for scheduled VMs, create a ConfigMap with a list of obsolete CPU models. When the `node-labeller` obtains the list of obsolete CPU models, it eliminates those CPU models and creates labels for valid CPU models.
+
+[NOTE]
+====
+If you do not configure a ConfigMap with a list of obsolete CPU models, all CPU models are evaluated for labels, including obsolete CPU models that are not present in your environment.
+====
+
+Through the process of iteration, the list of base CPU features in the minimum CPU model are eliminated from the list of labels generated for the node. For example, an environment might have two supported CPU models: `Penryn` and `Haswell`.
+
+If `Penryn` is specified as the CPU model for `minCPU`, the `node-labeller` evaluates each base CPU feature for `Penryn` and compares it with each CPU feature supported by `Haswell`. If the CPU feature is supported by both `Penryn` and `Haswell`, the `node-labeller` eliminates that feature from the list of CPU features for creating labels. If a CPU feature is supported only by `Haswell` and not by `Penryn`, that CPU feature is included in the list of generated labels. The `node-labeller` follows this iterative process to eliminate base CPU features that are present in the minimum CPU model and create labels.
+
+The following example shows the complete list of CPU features for `Penryn` which is specified as the CPU model for `minCPU`:
+
+.Example output
+[source,xml]
```

Comment:
Shouldn't use `source,xml` unless this is actually XML.

Same for the below code blocks.

Reply:
SME had confirmed it was xml, therefore used the xml formatting.

Reply:
@lmandavi that's not XML. XML would look something like:

```
<feature>apic</feature>
<feature>clflush</feature>
...
```
Suggest either just removing the source specification, or tagging the SME in this conversation to confirm what they meant when they said it was XML.

Reply:
Agree about the official xml formatting and how it is presented. In the conversations with @ksimon1, it was agreed to present the cpu feature names without the xml formatting - which is what is displayed currently. I will remove the source specification in the .adoc file for this display of CPU features.